tensorflow/python/distribute/cluster_resolver/slurm_cluster_resolver.py

```diff
@@ -27,6 +27,44 @@
 from tensorflow.python.training.server_lib import ClusterSpec
 from tensorflow.python.util.tf_export import tf_export
 
+from hostlist import expand_hostlist, parse_slurm_tasks_per_node
+
+
+def get_slurm_step_var(name):
+  """Get the SLURM step variable from the environment
+
+  Args:
+    name: Name of the step variable
+  Returns:
+    SLURM_STEP_<name> from os.environ
+  Raises:
+    RuntimeError if variable is not found
+  """
+  name = 'SLURM_STEP_' + name
+  try:
+    return os.environ[name]
+  except KeyError:
+    raise RuntimeError('%s not found in environment. Not running inside a SLURM step?' % name)
```

Comment:
Python max line length is 80.

Reply:
IIRC Python line length is even 79. Is there some style file or formatter for TF?

Reply:
E.g. the existing imports are already 93 chars long

Reply:
Imports are excluded from the line-length requirement. Here is the standard Google Python style guide, which we follow: http://google.github.io/styleguide/pyguide.html and there are some links to a formatter and vim settings file.